### ISSUE TITLE
Beta Checklist

### ISSUE THREAD

**Issue by ranger-ross** [ranger-ross/nihongo-stats#253]
## What?

**This issue serves as a tracker for what functionality needs to be completed before moving from the Alpha stage to Beta.**

## Goals
- Build out more "Overview" graphs
- Build our more BunPro functionality
- Bug fix/polish the UX of the BunPro and Anki to make them more useful
- Consider migrating to NextJS
- Better Error handling

## Issues
- #252 
- #239
- #251
- (on hold) #134
- #162
- #263
- #301
- #366

## Releasing
Once the tickets above are completed, this issue will be used to track the changing of the alpha label to beta
